src/Processing.js

```diff
@@ -3235,7 +3235,7 @@
         b = p.lerp(hsb1[2], hsb2[2], amt);
         rgb = p.color.toRGB(h, s, b);
         // ... and for Alpha-range
-        a = p.lerp(a1, a2, amt) * colorModeA;
+        a = p.round(p.lerp(a1, a2, amt) * colorModeA);
```

Comment:
That's pretty much as fast as it's going to get. It's already pretty much pointing to a native ternary macro